synapse/lib/schemas.py

```diff
@@ -173,6 +173,7 @@
         'creator': {'type': 'string', 'pattern': s_config.re_iden},
         'created': {'type': 'integer', 'minimum': 0},
         'nomerge': {'type': 'boolean'},
```

Comment:
Once migrated, should `nomerge` still be valid in the schema? Seems more like it should be converted automatically in the top-half of the nexus handlers if present?

Reply:
Removed.